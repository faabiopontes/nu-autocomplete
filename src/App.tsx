import GitHubAutocomplete from "./components/GitHubAutocomplete";
import GlobalStyle from './styles/global';
import AppProvider from './hooks/index';


const App = () => (
  <AppProvider>
    <h1>Welcome to NUAutocomplete</h1>
    <p>Below you should type any term and it will be searched through all React GitHub issues</p>
    <p>When the text field is selected you can use the <b>UP</b> or <b>DOWN</b> keyboard keys to go through the results shown</p>
    <p>By clicking or pressing <b>ENTER</b> when a specific issue is selected a new tab will open with their page</p>
    <GitHubAutocomplete />

    <GlobalStyle />
  </AppProvider>
);

export default App;
