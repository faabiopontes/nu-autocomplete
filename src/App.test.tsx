import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  it("should be rendered", () => {
    const component = render(<App />);
    expect(component).toMatchSnapshot();
  });
});
