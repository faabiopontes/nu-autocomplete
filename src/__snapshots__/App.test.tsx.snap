// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should be rendered 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1>
        Welcome to NUAutocomplete
      </h1>
      <p>
        Below you should type any term and it will be searched through all React GitHub issues
      </p>
      <p>
        When the text field is selected you can use the UP and DOWN keyboard keys to go through the results shown
      </p>
      <p>
        By clicking or pressing ENTER when a specific issue is selected a new tab will open with their page
      </p>
      <input
        class="sc-iCoGMd bcqdft"
        placeholder="Type here to get React issues"
        type="text"
        value=""
      />
      
      <div
        class="sc-bdnxRM donqmm"
      />
    </div>
  </body>,
  "container": <div>
    <h1>
      Welcome to NUAutocomplete
    </h1>
    <p>
      Below you should type any term and it will be searched through all React GitHub issues
    </p>
    <p>
      When the text field is selected you can use the UP and DOWN keyboard keys to go through the results shown
    </p>
    <p>
      By clicking or pressing ENTER when a specific issue is selected a new tab will open with their page
    </p>
    <input
      class="sc-iCoGMd bcqdft"
      placeholder="Type here to get React issues"
      type="text"
      value=""
    />
    
    <div
      class="sc-bdnxRM donqmm"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
